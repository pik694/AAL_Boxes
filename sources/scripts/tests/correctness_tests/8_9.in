8
81 56 83
63 35 60
77 30 70
72 55 12
90 47 4
6 67 87
14 9 53
11 37 13

