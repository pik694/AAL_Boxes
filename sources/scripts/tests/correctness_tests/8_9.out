7 3 2 0 6 1 4 5 
781871->560682
